docs/src/app/components/pages/components/Stepper/CustomStepConnector.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import {
+  Step,
+  Stepper,
+  StepLabel,
+} from 'material-ui/Stepper';
+import ArrowForwardIcon from 'material-ui/svg-icons/navigation/arrow-forward';
+
+/**
+ * It is possible to specify your own step connector by passing an element to the `connector`
+ * prop. If you want to remove the connector, pass `null` to the `connector` prop.
+ */
+class CustomStepConnector extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleNext = this.handleNext.bind(this);
+    this.handlePrev = this.handlePrev.bind(this);
+  }
+
+  state = {
+    stepIndex: 0,
+  };
+
+  getStepContent(stepIndex) {
+    switch (stepIndex) {
+      case 0:
+        return (
+          <p>
+            For each ad campaign that you create, you can control how much you're willing to
```

Comment:
That ugly GitHub code highlighting can be fixed using a string element: `{"For each ad..."}`.